packages/launch/src/stores/index.ts

```diff
@@ -0,0 +1,12 @@
+// this file is a copy of apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/stores/index.ts
+// TODO: avoid code duplication
```

Comment:
TODO: remove code duplication between `packages/launch/src/stores` and `apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/stores`

Reply:
This is resolved in https://github.com/Automattic/wp-calypso/pull/46834.